src/app/components/pages/MyAccount.tsx

```diff
@@ -377,7 +380,7 @@ const AccountPageComponent = ({user, getChosenUserAuthSettings, errorMessage, us
                                     {/* Teacher connections does not have a save */}
                                     <Input
                                         type="submit" value="Save" className="btn btn-block btn-secondary border-0"
-                                        disabled={!accountInfoChanged || activeTab === ACCOUNT_TAB.teacherconnections}
+                                        disabled={!accountInfoChanged || activeTab === ACCOUNT_TAB.teacherconnections || !isNewPasswordConfirmed}
```

Comment:
Not totally clear on what's going on here - this function appears to be setting a new programming language, why is the password status being checked?

Reply:
This is a Save button the My Account Page, which has tabs for Profile, Account Security, Teacher connections, Email preferences, Beta features. The Save button applies to all the tabs except Teacher connections (hence that limitation). I added the password check, but that it will only block the save button if they've filled out the old password and the new password, but the new & confirmation PW don't match.  If they haven't gone onto the Account Security tab and changed the PW fields, the Save button works fine on other tabs.